### PR TITLE
adding the setNotifications function

### PR DESCRIPTION
**Pull request by filipeotero**
In this PR, instead of requesting the back-end API, we insert the notifications as a parameter to the notifications panel. 
Also, the notifications-panel is updated according to dynamods organization.

FYI: @QilongTang

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,51 +1,49 @@
...
-              <a href={notifications[i].link} target="_blank">{notifications[i].linkTitle}</a>
-            </div>
-          };
-          notificationData.push(notificationItem);
-        }
-        setAPIData(notificationData);
-      });
+
+  const setNotifications = (notifications) => {
+    console.log(notifications)
```
Comment (avidit) [inline src/App.js:14]: can we remove this line?

File: src/App.js [hunk 2]
```diff
@@ -1,51 +1,49 @@
 import './App.css';
 import axios from 'axios';
```
Comment (avidit) [inline src/App.js:2]: Do you we will still need axios package ?

Comment (filipeotero) [reply src/App.js]: No need, removed.

**Comment by QilongTang** [comment]
@filipeotero It looks like after your change, launching the webApp alone, folks can no longer test the display of message from our backend endpoint. Anyway we can preserve both options?

**Comment by filipeotero** [comment]
> @filipeotero It looks like after your change, launching the webApp alone, folks can no longer test the display of message from our backend endpoint. Anyway we can preserve both options?

We can set an env variable that gets the notifications by itself or waits to get the notifications from the `setNotifications ` function

File: package.json [hunk 3]
```diff
@@ -32,7 +32,7 @@
     "@babel/core": "^7.18.6",
     "@babel/preset-env": "^7.18.6",
     "@babel/preset-react": "^7.18.6",
-    "@filipeop/notifications-panel": "^0.0.2",
+    "@dynamods/notifications-panel": "^0.0.2",
```
Comment (QilongTang) [inline package.json:35]: 👍🏻

**Comment by filipeotero** [comment]
Hi @QilongTang, notice that in this new commit I added a verification before fulfilling the notifications list. If the env URL is empty, we set the `setNotifications `function from outside the webApp. Otherwise, we grab the notifications by the backend.

**Comment by QilongTang** [comment]
> Hi @QilongTang, notice that in this new commit I added a verification before fulfilling the notifications list. If the env URL is empty, we set the `setNotifications `function from outside the webApp. Otherwise, we grab the notifications by the backend.

In this case, you probably want to move the value in https://github.com/DynamoDS/NotificationCenter/blob/master/config/.env to .env.dev so that when this package is bundled in prod, the WebApp is used for purely display. But when building it locally, it will grab notifications from dev end point. What do you think @zeusongit ?

**Comment by avidit** [comment]
> In this case, you probably want to move the value in [https://github.com/DynamoDS/NotificationCenter/blob/master/config/.env](https://github.com/DynamoDS/NotificationCenter/blob/master/config/.env?rgh-link-date=2022-08-23T15%3A56%3A22Z) to .env.dev so that when this package is bundled in prod and used inside Dynamo, the WebApp is used for purely displaying notifications passed in by Dynamo. But when building it locally, it will grab notifications from dev end point. What do you think @zeusongit ?

@QilongTang I have made these changes and updated package.json with --env parameters in https://github.com/DynamoDS/NotificationCenter/pull/23/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R24-R25

**Comment by zeusongit** [comment]
> > Hi @QilongTang, notice that in this new commit I added a verification before fulfilling the notifications list. If the env URL is empty, we set the `setNotifications `function from outside the webApp. Otherwise, we grab the notifications by the backend.
> 
> In this case, you probably want to move the value in https://github.com/DynamoDS/NotificationCenter/blob/master/config/.env to .env.dev so that when this package is bundled in prod and used inside Dynamo, the WebApp is used for purely displaying notifications passed in by Dynamo. But when building it locally, it will grab notifications from dev end point. What do you think @zeusongit ?

Makes sense, with @avidit's changes we will get different URL's based on the environment used.

**Comment by QilongTang** [comment]
Thanks, with the other PR in progress, is this one good to go? @zeusongit @avidit

**Comment by zeusongit** [comment]
Looks good, just one comment, maybe unrelated to this PR, but we include this script tag:
https://github.com/DynamoDS/NotificationCenter/blob/71826235f9d2649477281c1ada4cadb4d2f7e21b/src/index.html#L13
which is understandable if we are working on notification center alone but inside dynamo `mainJs` is replaced by the same file we are including from the script tag, therefore are we including the bundle file twice, when running inside Dynamo?

**Review by zeusongit — APPROVED** [review]

**Comment by QilongTang** [comment]
> 

Ah this is something I forgot to mention to you, when launching in Dynamo, <script src="index.bundle.js"></script>  will fail to load. This only works for dev build here in this repo. Maybe we should use different html for differernt build instead of mixing things..

**Pull request merged**